src/Infrastructure/Event/CliRunnerSetupListener.php

```diff
@@ -16,7 +16,7 @@ class CliRunnerSetupListener
     protected LocalCliRunner $cliRunner;
 
     /**
-     * @param \SprykerSdk\Sdk\Infrastructure\Service\LocalCliRunner $cliRunner
+     * @param \SprykerSdk\Sdk\Infrastructure\Service\CommandRunner\LocalCliRunner $cliRunner
```

Comment:
It must be depended on the abstraction. Please change it to the `CommandRunnerInterface` interface.